ducktape/tests/loader.py

```diff
@@ -26,32 +27,103 @@ class LoaderException(Exception):
     pass
 
 
+# A helper container class
+ModuleAndFile = collections.namedtuple('ModuleAndFile', 'module file')
```

Comment:
Interesting, I didn't know you could do `'module file'` instead of `['module', 'file']`